nifi-nar-bundles/nifi-kafka-bundle/nifi-kafka-2-6-processors/src/main/java/org/apache/nifi/processors/kafka/pubsub/PublishKafkaRecord_2_6.java

```diff
@@ -271,7 +273,21 @@ public class PublishKafkaRecord_2_6 extends AbstractProcessor implements Verifia
         .defaultValue("UTF-8")
         .required(false)
         .build();
-
+    static final PropertyDescriptor PUBLISH_STRATEGY = new PropertyDescriptor.Builder()
+            .name("publish-strategy")
+            .displayName("Publish Strategy")
+            .description("The format used to publish the outgoing FlowFile record to Kafka.")
+            .required(true)
+            .defaultValue(WRITE_VALUE_ONLY.getValue())
+            .allowableValues(WRITE_VALUE_ONLY, USE_WRAPPER)
```

Comment:
I'm not sure that "Write Value Only" is applicable here. WIth this strategy, we can still send message headers using attributes, and a message key by specifying a field name.

Reply:
I understand you're suggesting a different name.  What would that be?

option:
- WRITE_VALUE, WRITE_WRAPPER

Or are you saying there shouldn't be an option?

Reply:
Yes, sorry, I was suggesting renaming it. Perhaps something like "Use Content as Record Value" or "FLowFile Content as Record Value" or something like that? Was just trying to convey that we aren't *only* writing the record value, a key and headers may also be written.